docs/environment.yml

```diff
@@ -1,7 +1,8 @@
 name: voila
 channels:
   - conda-forge
-  - conda
+  - nodefaults
```

Comment:
I'm really surprised by this. Shouldn't we only use the `conda-forge` channel?

Reply:
No idea. It could have been a temporary issue on ReadTheDocs as well.

I also saw this change in https://github.com/jupyterlab/jupyterlab_server/pull/296